### DISCUSSION TITLE
slider velocity value uses comma as separator instead of dot

### DISCUSSION THREAD

**Discussion by Dopaminos** [ppy/osu#21373] [Q&A] [answered]
i'm mapping right now, and this issue is kinda annoying. 
when i'm trying to change sv, im just changing last digits of number for small changes. and because sv value being displayed with comma, sv goes 10x and i need to edit sv manually again but now with full number. that's just annoying, plz fix
(display sv with dot instead of comma)

https://user-images.githubusercontent.com/42616718/203269998-52473d81-e61e-4afe-8511-c3d9213432b1.mp4

**Comment by peppy** [comment]
I feel like this is being tracked in an issue thread already, but this really does deserve p0 attention, very silly.

**Reply by Dopaminos** [reply]
i'd add it to #12020 tbh

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/20989
